pkg/distributor/otel.go

```diff
@@ -203,6 +291,7 @@ func OTLPHandler(
 		level.Debug(spanLogger).Log(
 			"msg", "OTLP to Prometheus conversion complete",
 			"metric_count", metricCount,
+			"metric_dropped", metricsDropped,
```

Comment:
Plural metrics are dropped:
```suggestion
			"metrics_dropped", metricsDropped,
```